R/MeasureFairness.R

```diff
@@ -0,0 +1,54 @@
+#' @title Fairness Measure
+#' @name mlr_measures_fairness
+#'
+#' @description
+#' This measure specializes [Measure] for Fairness Measure problems:
+#' Users could use [groupwise_abs_diff()], [groupwise_diff()], or [groupwise_quotient()] to evaluate
+#' the fairness measures based on base_measures. For example, the false positive rate bias or
+#' the equalized odds ratios. More examples could viewed on mlr3fairness wikipage.
+#'
+#' Predefined measures can be found in the [dictionary][mlr3misc::Dictionary] [mlr_measures].
+#' Predefined operations can be found in the [mlr3fairness].
+#'
+#' @export
+MeasureFairness = R6Class("MeasureFairness", inherit = Measure, cloneable = FALSE,
+  public = list(
+    #' @template field_fun
+    fun = NULL,
```

Comment:
Not clear what this is supposed to be.

Reply:
Those are the templates stored at man-roxygen. I followed the documentation styles in mlr3.

Reply:
Should I remove them?

Reply:
I think If you do not need/use the slot just get rid of it yes.

Reply:
Sorry, my comment was not clear. I was wondering what the public field `fun` is supposed to store. The documentation just states `"This function referred to the fairness measure operation functions."`. I don't think that this is helpful for the user. Maybe this field should be private?

Reply:
I see what you mean. I guess I will change this field to private.

Reply:
Also this field is needed to store the operation function, so we cannot get rid of it.